lambda_functions/archive_service/get_closed_items.py

```diff
@@ -15,10 +15,22 @@ def get_closed_items(event, context):
     helper.log_method_initiated("Get all closed items", event, logger)
 
     with Session() as session:
+        try:
+            url = event["queryStringParameters"]['url']
+            logger.info("Found url in queryparams")
+        except:
+            url = None
+            logger.info("No url found in queryparams")
+            pass
```

Comment:
Ich persönlich finde es nicht so schön, dass hier mittels `try except` geprüft wird, welche handler Methode aufgerufen werden soll. So wird ja bei jedem Aufruf, der keine URL enthält eine Exception forciert.


Ich würde den Block mit dem `if else` block ab Zeile 30 verbinden.

Sprich:

```
if "queryStringParameters in event and "url" in event["queryStringParameters"]:
  items = ...
else: 
  items = ...```

Reply:
Ist angepasst in #373